docs/source/roadmap.rst

```diff
@@ -16,34 +16,30 @@ contributions. Here's our plans for the next two releases.
     the feature. Pull Requests are open to anyone.
 
 
-3.1
```

Comment:
As we released `3.1`, the implemented items from that release needs be moved under `Release History`

Reply:
Done